config/example_config.yml

```diff
@@ -251,26 +252,33 @@ Model:
     # Required dictionary containing a module/function pair.
     # Module in model directory and function in module implementing the model.
     # Module and function pairs included with CTLearn:
-    #   - {module: 'single_tel', function: 'single_tel_model'}: single
-    #       tel model using a basic convolutional neural network (CNN)
+    #   - {module: 'single_cnn', function: 'single_cnn_model'}: single
+    #       telescope model using a basic convolutional neural network (CNN)
+    #   - {module: 'res_net', function: 'res_net_model'}: Residual block
+    #       model using a basic convolutional neural network (CNN)
     #   - {module: 'cnn_rnn', function: 'cnn_rnn_model'}: array model
     #       feeding output of a basic CNN for each  telescope into a
     #       recurrent neural network (RNN)
     #   - {module: 'variable_input_model', function: 'variable_input_model'}:
     #       array model feeding combined outputs of a CNN for each
     #       telescope into a CNN network head
-    model: {module: 'single_tel', function: 'single_tel_model'}
+    model: {module: 'single_cnn', function: 'single_cnn_model'}
 
-    # Dictionary of labels where the keys are the label names.
-    # For a classification label, the value is a list of class names
+    # Dictionary of tasks where the values contain the fully connected layers
+    # head of each task. The final loss is calculated with a weighted sum 
+    # over the individual losses. Currently the weights are constanst values.
+    # For the classification task, the class names are passed into the dict
     # in order of class label (0 to n-1).
-    # For a regression label, the value is null.
     # Optional dictionary with string keys and values either null or
     # a list of strings. Default: {}
-    label_names:
-        class_label:
-            - 'proton'
-            - 'gamma'
+    # These options have to be in consonance with the 'event_info' and 
+    # 'transforms' in 'Data'.
```

Comment:
It would helpful to document what the necessary transforms and event_info are for each task.

Reply:
Done